Fakebook.Profile/Fakebook.Profile.Domain/DomainProfile.cs

```diff
@@ -212,6 +238,9 @@ public DomainProfile(string email, string firstName, string lastName)
             FirstName = firstName;
             LastName = lastName;
             ProfilePictureUrl = new Uri(ProfileConfiguration.DefaultUri);
+            FollowerEmails = new List<string>();
+            FollowingEmails = new List<string>();
```

Comment:
FYI this code also has no reason to be in this PR. with TDD phase 1, literally the only code you add that's not tests is the minimum needed to get the tests compiling and failing. but i'll let it slide this time.